src/components/Router/StepsRouter.js

```diff
@@ -65,7 +65,13 @@ class StepsRouter extends Component {
           })}
         >
           <CurrentComponent
-            {...{ ...options, ...globalUserOptions, ...otherProps, back }}
+            {...{
+              ...options,
+              ...globalUserOptions,
+              ...otherProps,
+              mobileFlow,
+              back,
+            }}
```

Comment:
I'm against this usage, nested rest operators seem confusing especially in several lines.
```suggestion
          const passedProps = {
              ...options,
              ...globalUserOptions,
              ...otherProps,
              mobileFlow,
              back,
            }
          {...passedProps}
```

Reply:
Great suggestion. Think part of why this property got missed out was from the use of nested rest operators around the code here in general.